charters/DataProcessingService/DataProcessingService.md

```diff
@@ -0,0 +1,51 @@
+# [Data Processing Service](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+The data processing service is a service that executes automated transformation to scientific data through cloud-based execution of scientific pipelines. It can be used in any instantiation of the Data Coordination Platform (DCP). 
+
+## Objectives
+The data processing service aims to develop the infrastructure to run transformative scientific analysis pipelines reliably and at-scale for an instantiation of the DCP. The team working on the service will also operate the infrastructure for the Human Cell Atlas. 
+
+## In-scope
+* Creating and maintaining the Data Processing Service
+* Operating the instantiation of the HCA DCP Data Processing Service  
+* Initiation and management of the execution of all data processing pipelines that run in the HCA DCP  
+* Documentation on the Data Processing Service and guides on using the service 
+* Establishing and operating automated practices around engineering testing of pipelines in the HCA DCP 
+* Creating and outputting metadata on the transformations done to the data in accordance with the Metadata Schema
+
+## Out-of-scope
+* Standing up or operating other instantiations of the Data Processing Service
```

Comment:
I assume this is because you will operate the version of the data processing service for the HCA instance of the DCP but not anyone else's data processing service using the code created by this service.

I think the language throughout the charter could be clearer. This is the first time such an explicit statement is made whereas elsewhere it feels more implied.